style.css

```diff
@@ -18,9 +18,36 @@ h3, p {
   font-family: "Skia", "Open Sans", sans-serif
 }
 
+.body-image{
+  width: 30%;
+  min-width: 200px;
+  display: flex;
+  float: left;
+}
+
+
+.body-table{
+  width: 40%;
+  float: center;
+  padding-left: 50%
+  padding-right: 50%;
+}
```

Comment:
You got a few errors in this block. I don't think `center` is a valid value for `float` and you're missing a semi-colon on `padding-left` line.